anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -420,11 +417,12 @@ def __init__(
             raise ValueError(
                 "radial_gaussian_width is set as an integer, which could cause overflow errors. Pass in float."
             )
-        radial_hypers = {}
-        radial_hypers["radial_basis"] = radial_basis_name.lower()  # lower case
-        radial_hypers["radial_gaussian_width"] = radial_gaussian_width
-        radial_hypers["max_angular"] = max_angular
-        self.radial_basis = RadialBasis(**radial_hypers)
+        radial_hypers = {
+            "radial_gaussian_width": radial_gaussian_width,
+        }
+        self.radial_basis = RadialBasis(
+            radial_basis_name.lower(), max_angular, **radial_hypers
+        )
```

Comment:
If we want to remove the `radial_hypers` mid-step, we can just supply `radial_gaussian_width` as an explicit arg too:
```suggestion
        self.radial_basis = RadialBasis(
            radial_basis_name.lower(), max_angular=max_angular, radial_gaussian_width=radial_gaussian_width
        )
```

Reply:
I'd say let's keep the dict then! I like doing that over explicit kwargs

Reply:
agree with the dict